backend/tests/engine/projects/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from app.modules.dto import AllocationDTO
+
+
+@pytest.fixture
+def data_for_quadratic_funding(projects):
+    """
+    This fixture is used to create a list of allocations for the quadratic funding algorithm.
+
+    Data is got from Octant's Excel file: https://docs.google.com/spreadsheets/d/1GpZ5ZIJX8nYh7xLw9P9lIoXIVAxiMgda89sN3iYFFiI/edit#gid=0
```

Comment:
Can you also test with another test set? That way we check the calculations based on some external data. You can find it online or just create yourself. It can be a separate task.

Reply:
I think it makes profound sense to put the data directly in the docstr as not to be linked to an external source (and I did so). But creating another test set doesn't change anything else since already used data is various enough and aligns our case fair enough. Please take a look after changes